app-misc/Clipboard/files/Clipboard-0.6.0-libdestination-and-disable-lto-and-git.patch

```diff
@@ -54,9 +54,25 @@ diff -ur '--color=auto' Clipboard-0.6.0/src/clipboard/CMakeLists.txt Clipboard-0
  target_link_libraries(clipboard gui)
  
  if(WIN32)
+@@ -45,13 +43,13 @@
+     TARGET clipboard
+     APPEND
+     PROPERTY BUILD_RPATH
+-    "${CMAKE_INSTALL_PREFIX}/lib"
++    "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+   )
+   set_property(
+     TARGET clipboard
+     APPEND
+     PROPERTY INSTALL_RPATH
+-    "${CMAKE_INSTALL_PREFIX}/lib"
++    "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
```

Comment:
Bad idea, `CMAKE_INSTALL_LIBDIR` may be absolute. Use `CMAKE_INSTALL_FULL_LIBDIR` exclusively instead.